keyboards/kbdfans/kbd67/rev2/keymaps/tucznak/config.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2018 MechMerlin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
+
+#define TAPPING_TERM 50
+#define RETRO_TAPPING
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_SLEEP
+#endif
+
+#ifdef MANUFACTURER
```

Comment:
you could remove the `ifdef`'s for these. They should always be defined.  

It's not necessary, but would make it a bit cleaner looking.

Reply:
Okay, if I don't have to check and MANUFACTURER, PRODUCT, DESCRIPTION are always defined, I'll just straight up undef them and replace with my own.
Alternatively, not sure if this is frowned upon or not, but I could just remove the replacement of those values entirely and just stick with the default values. It was mostly a novelty anyway, but I find that Windows doesn't really display these anywhere meaningful.

Reply:
Well, I'm doing the same thing. So ....  But yeah, I just undef. 

As for Windows, it does, if you look in the Windows 10 Settings -> Devices, or if you look in the Control Panel, under "Hardware and Sound" -> "Devices and Printers"